From 08571bf8d4ba1e7546b46e7d24011ca2f9be1f3d Mon Sep 17 00:00:00 2001
From: David Pacheco <dap@joyent.com>
Date: Tue, 14 Feb 2017 11:21:39 -0800
Subject: [PATCH] joyent/mdb_v8#85 missing dependency for mdb_v8_version.c
 leads to build failure Reviewed by: Joshua M. Clulow <jmc@joyent.com>
 Approved by: Joshua M. Clulow <jmc@joyent.com>

---
 GNUmakefile | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/GNUmakefile b/GNUmakefile
index 173f325..21453d3 100644
--- a/GNUmakefile
+++ b/GNUmakefile
@@ -165,9 +165,12 @@ include Makefile.arch.targ
 # is baked into the binary itself (so the dmod can report its version) and also
 # used in the publish target.
 #
-$(MDBV8_BUILD)/mdb_v8_version.c: version
+$(MDBV8_BUILD)/mdb_v8_version.c: version | $(MDBV8_BUILD)
 	tools/mkversion < $^ > $@
 
+$(MDBV8_BUILD):
+	$(MKDIRP)
+
 #
 # Include common Joyent Makefile for JavaScript "check" targets.
 #
-- 
2.21.0

